app/controllers/event_session_comments_controller.rb

```diff
@@ -0,0 +1,44 @@
+class EventSessionCommentsController < ApplicationController
+  before_filter :authenticate_user!
+
+  def create
+    @event_session_comment = EventSessionComment.new(comment_params)
+
+    @event_session_comment.save
```

Comment:
Test for DB persistence to be successful or error and response appropriately. Check this as an example:

https://github.com/null-open-security-community/swachalit/blob/master/app/controllers/session_proposals_controller.rb